.spin/cmds.py

```diff
@@ -86,15 +70,35 @@ def asv(asv_args):
     util.run(['asv'] + list(asv_args))
 
 
-@click.command()
-def coverage():
-    """📊 Generate coverage report
-    """
-    util.run(['python', '-m', 'spin', 'test', '--', '-o', 'python_functions=test_*', 'skimage', '--cov=skimage'], replace=True)
-
-
 @click.command()
 def sdist():
     """📦 Build a source distribution in `dist/`.
     """
     util.run(['python', '-m', 'build', '.', '--sdist'])
+
+
+@click.command(context_settings={
+    'ignore_unknown_options': True
+})
+@click.argument("ipython_args", metavar='', nargs=-1)
+@click.pass_context
+def ipython(ctx, ipython_args):
```

Comment:
Wouldn't it be better / possible to wrap spin's ipython command with `ctx.forward(meson.ipython) and add the `--TerminalIpython...` bit to `ipython_args` instead of re-implementing it?

Reply:
It's a very short function, but that would be an option.

Reply:
I refactored it as you requested.

Reply:
Thanks.